LVT.Tests/EpicPresenterTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using LVT.LVT.Services;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LVT.Tests
+{
+    class EpicPresenterTests
+    {
+        [Test]
+        public void EpicPresenterTest()
+        {
+            var EP = new EpicPresenter();
+            var epic = new Epic("testDescription", "testDeadline");
+            string result = EP.VisualizeToString(epic, "parentNode");
+            string NodeID = epic.NodeID;
+            string Description = epic.Description;
+            string Deadline = epic.Deadline;
+            string ParentNode = "parentNode";
+            string expected = "[{ v: '" + NodeID + "', f: 'Epic" + "<div style=\"font-style:italic\">" + $"{Description}" + "</div>" +
+                                                                   "<div style=\"font-style:italic\">" + $"{Deadline}" + "</div>" +
+                                                                   "'}, " + $"'{ParentNode}']";
```

Comment:
This can be broken down into smaller tests, look at testing each part of the string individually

Reply:
I may have misunderstood, but I think this would have applied had we broken down the 'string conversion' into smaller parts the way Kieron suggested earlier (like have one class that produces text in italic, one class that produces <div></div> elements with the text you give it etc).  For here, I'm not so sure you can reasonably break it down into smaller pieces, because the method you're calling actually returns the entire string as is.

Reply:
As discussed we'll take a look at extracting into a base class that's tested, with each of the Presenter classes then inheriting from this class. That way we're not having to copy and paste as much, then if we end up refactoring into separate classes (e.g. to supply the `style` or `div`) we only have to look at the tests on the base class.

Reply:
@victorjefferies and I will take a look at that